### DISCUSSION TITLE
What is the purpose of config namespace?

### DISCUSSION THREAD

**Discussion by cduff** [facebook/lexical#2661] [Q&A]
I'm experimenting with Lexical in React as per the [getting started guide](https://lexical.dev/docs/getting-started/react). The config requires a `namespace` value to be specified but I don't know why and what this value is used for. I can't find any info around this. Can anyone provide some insight on this? I'm currently just using an empty string which is working fine.

**Comment by maltesa** [comment]
I'm wondering the same. I found out that I'm able to copy/paste decorator nodes (e.q. equations) between multiple editor instances if the namespace is equal. If I'm using different namespaces between editor instances, the decorator nodes are copied as plain text.

Hence, my guess at the moment is: *Namespaces are for marking editor instances as compatible?*

**Question:** Are there any downsides to using the same namespace (given plugins between instances are equal)?

**Reply by evbo** [reply]
Thanks for sharing this observation. The only mention of it in the API docs I could find confirms what you wrote - it has something to do with clipboard?
https://lexical.dev/docs/api/modules/lexical_clipboard#returns
